saleor/product/migrations/0183_calculate_discounted_prices.py

```diff
@@ -289,10 +289,10 @@ def _get_product_to_variant_channel_listings_per_channel_map(
     }
 
     price_data = defaultdict(lambda: defaultdict(list))
-    for variant_channel_listings in variant_channel_listings:
+    for variant_channel_listing in variant_channel_listings:
         product_id = variant_to_product_id[variant_channel_listings.variant_id]
         price_data[product_id][variant_channel_listings.channel_id].append(
```

Comment:
```suggestion
        product_id = variant_to_product_id[variant_channel_listing.variant_id]
        price_data[product_id][variant_channel_listing.channel_id].append(
```
Please check it, but I believe we should change here as well